src/main/webapp/view/transcript/transcript-area.js

```diff
@@ -72,15 +75,32 @@ export default class TranscriptArea {
     ulElement.class = 'mx-auto';
     transcriptContainer.appendChild(ulElement);
     transcriptLines.forEach((transcriptLine) => {
-      ulElement.appendChild(
-          TranscriptLineElement.createTranscriptLineElement(transcriptLine));
+      const transcriptLineElement =
+          TranscriptLineElement.createTranscriptLineElement(transcriptLine);
+      ulElement.appendChild(transcriptLineElement);
+      TranscriptArea.#transcriptLineMap.set(
+          transcriptLine.transcriptKey.id, transcriptLineElement);
     });
   }
 
+  /** Increments the indicator corresponding to `transcriptLineKey` by 1. */
+  static incrementCommentIndicatorAt(transcriptLineKeyId) {
```

Comment:
where does this method get called?

Reply:
This method gets called in `DiscussionArea` as the new comments are inserted so I would have to pass a reference of `TranscriptArea` to `DiscussionArea`

Reply:
Why is the variable called transcriptLineKeyId and not transcriptLineKey? This does not match the JsDoc.

Reply:
ahh, thanks for catching that!

I updated it to be `transcriptLineKeyId` because a transcriptLine key is formatted like this: {kind: "TranscriptLine", id: 123456}. It is easier to just hold onto the id since we know the kind is `transcriptLine`.